homeassistant/components/trafikverket_weatherstation/config_flow.py

```diff
@@ -51,7 +48,7 @@ async def async_step_user(self, user_input=None):
             name = user_input[CONF_NAME]
             api_key = user_input[CONF_API_KEY]
             station = user_input[CONF_STATION]
-            conditions = user_input[CONF_MONITORED_CONDITIONS]
+            conditions = json.dumps(list(SENSOR_LIST))
```

Comment:
Why are we storing this in the config entry if it's just a list of all sensor types? That's not needed.